tests/unit/transport/test_zeromq.py

```diff
@@ -305,3 +305,34 @@ def test_send(self):
     def test_destroy(self):
         self.message_client_pool.destroy()
         self.assertEqual([], self.message_client_pool.message_clients)
+
+
+class ZMQConfigTest(TestCase):
+    def test_master_uri(self):
+        '''
+        test _get_master_uri method
+        '''
+        m_ip = '127.0.0.1'
+        m_port = 4505
+        s_ip = '111.1.0.1'
+        s_port = 4058
+
+        # source ip and source_port empty
+        assert salt.transport.zeromq._get_master_uri(master_ip=m_ip,
```

Comment:
Why `assert` and not `self.assertEqual` is here? The difference in the output if a failure case will be sensitive, for instance:
```
AssertionError
```
vs
```
AssertionError: u'A brown fox jumped over the red hen.' != u'Abrownfoxjumpedovertheredhen.'
       - A brown fox jumped over the red hen.
       ?  -     -   -      -    -   -   -
       + Abrownfoxjumpedovertheredhen.
```

Reply:
`assert` is the syntax pytest uses and `self.assertEqual` is the syntax `unittest2` uses. We can support both ways. It is true that the unittest2 syntax makes it easier to read the failures, but since we will be moving towards pytest in the future, I am OK with this as-is.